frontend/app/(no-footer)/organiser/dashboard/page.tsx

```diff
@@ -71,13 +71,11 @@ export default function Dashboard() {
                 </div>
               </div>
             </div>
-            <div className="overflow-auto">
-              <div className="bg-organiser-light-gray py-4 rounded-2xl">
-                <h1 className="text-2xl font-bold text-center w-full sm:w-[300px] xl:w-[290px] 2xl:w-[320px]">
-                  Upcoming Events
-                </h1>
+            <div className="md:overflow-auto mt-6 lg:mt-0">
+              <div className="bg-organiser-light-gray py-4 rounded-2xl lg:px-8">
+                <h1 className="text-2xl font-bold text-center w-full">Upcoming Events</h1>
               </div>
-              <div>
+              <div className="sm:grid sm:grid-cols-2 sm:gap-2 lg:block">
                 {eventDataList
```

Comment:
the eventDataList get chopped off at the bottom in mobile view:
![Uploading image.png…]()